pkg/karmadactl/util/cluster.go

```diff
@@ -95,6 +96,19 @@ func DeleteClusterObject(controlPlaneKubeClient kubeclient.Interface, controlPla
 	return err
 }
 
+// GetClusterSyncMode return the sync mode of the member cluster.
+func GetClusterSyncMode(controlPlaneKarmadaClient karmadaclientset.Interface, clusterName string) (clusterv1alpha1.ClusterSyncMode, error) {
```

Comment:
In my opinion, this function doesn't feel very necessary, and inconvenient to test